infra-as-code/bicep/modules/privateDnsZones/parameters/privateDnsZones.parameters.all.json

```
@@ -86,6 +86,9 @@
     },
     "parVirtualNetworkIdToLink": {
       "value": "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/xxxxxxxxxxx/providers/Microsoft.Network/virtualNetworks/xxxxxxxxxxx"
+    },
+        "parVirtualNetworkIdToLinkFailover": {
+      "value": "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/xxxxxxxxxxx/providers/Microsoft.Network/virtualNetworks/xxxxxxxxxxx"
```

```suggestion
      "value": ""
```

default to empty as not a common architecture for most